bedrock/firefox/views.py

```diff
@@ -394,7 +394,7 @@ def get_template_names(self):
             template = 'firefox/dev-firstrun.html'
         elif show_40_firstrun(version):
             if locale == 'en-US' and exp in ['a', 'b']:
-                template = 'firefox/firstrun/index-{0}.html'.format(exp)
+                template = 'firefox/firstrun/membership-{0}.html'.format(exp)
```

Comment:
Note: unit tests are currently failing due to this. The previous experiment is going to be stopped & removed prior to this merging, so I'm just leaving it like this for now.